tslint.json

```diff
@@ -1,40 +1,30 @@
 {
+  "defaultSeverity": "error",
+  "extends": ["tslint:all", "tslint-config-prettier"],
   "rules": {
-    "file-name-casing": [true, "camel-case"],
-    "class-name": true,
-    "comment-format": [true, "check-space"],
-    "indent": [true, "spaces", 2],
-    "one-export": false,
-    "no-console": true,
-    "no-debugger": true,
-    "no-eval": true,
-    "no-internal-module": true,
-    "no-trailing-whitespace": true,
-    "no-unsafe-finally": true,
-    "no-var-keyword": true,
-    "one-line": [true, "check-open-brace", "check-whitespace"],
-    "quotemark": [true, "single", "avoid-escape"],
-    "semicolon": [true, "never"],
-    "triple-equals": [true, "allow-null-check"],
-    "typedef-whitespace": [
+    "completed-docs": [
       true,
       {
-        "call-signature": "nospace",
-        "index-signature": "nospace",
-        "parameter": "nospace",
-        "property-declaration": "nospace",
-        "variable-declaration": "nospace"
+        "interfaces": {
+          "visibilities": ["exported"]
+        },
+        "classes": {
+          "visibilities": ["exported"]
+        }
       }
     ],
-    "variable-name": [true, "ban-keywords"],
-    "whitespace": [
-      true,
-      "check-branch",
-      "check-decl",
-      "check-operator",
-      "check-separator",
-      "check-type"
+    "variable-name": [true, "allow-pascal-case"],
+    "typedef": [true, "call-signature"],
+    "file-name-casing": [true, "camel-case"],
+    "no-implicit-dependencies": [
+      false, // Doesn't play well with mono-repo
+      "dev"
     ],
-    "ordered-imports": true
+    "only-arrow-functions": false,
+    "increment-decrement": false,
+    "no-null-keyword": false,
+    "no-magic-numbers": false,
+    "no-parameter-properties": false,
+    "no-unused-expression": true
```

Comment:
Why are last 6 set to be false?

Reply:
- have activated `no-null-keyword`, and disabled it for test files
- have activated `no-magic-numbers` and disabled it for test files
- `no-parameter-properties` is set to `false` to allow constructor-param-initialization